script.py

```diff
@@ -3,31 +3,82 @@
 import os
 import time
 import logging
+import structlog
 import boto3
 from botocore.exceptions import ClientError
 from kubernetes import client, config
 
+AWS_REGION = os.environ['AWS_REGION']
+
+APPLICATION_NAME = 'aws-secrets-synchronizer'
+APPLICATION_VERSION =  os.environ['APPLICATION_VERSION'] # Mandatory, will raise KeyError if not set
+ENVIRONMENT_TYPE = os.environ['ENVIRONMENT_TYPE'] # Mandatory, will raise KeyError if not set
+ENVIRONMENT_NAME = os.environ.get('ENVIRONMENT_NAME')
+
+
+def get_base_logger(name=None):
+    structlog.configure(
+        processors=[
+            structlog.contextvars.merge_contextvars,
+            structlog.processors.add_log_level,
+            structlog.processors.StackInfoRenderer(),
+            structlog.dev.set_exc_info,
+            structlog.processors.EventRenamer("msg"),  # rename 'event' to 'msg'
```

Comment:
```suggestion
            structlog.processors.EventRenamer("message"),  # rename 'event' to 'msg'
```

use complete words 🙏